common/source/docs/common-understanding-altitude.rst

```diff
@@ -0,0 +1,82 @@
+hoving.. _common-understanding-altitude:
+
+===================================
+Understanding ALTITUDE in ArduPilot
+===================================
+
+ArduPilot documentation uses the word "altitude" often and in many different ways and contexts. Not all "altitudes" mean the same thing, and an good understanding of what altitude means in any given context is important to understand expected operation and behavior.
+
+
+.. image:: ../../../images/altitudes.jpg
+    :target: ../_images/altitudes.jpg
+
+
+Definitions of Altitude (ALT)
+=============================
+
+In the above diagram, various types of "altitude" are shown:
+
+- ASL: Altitude above (mean) Sea Level - the altitude of the vehicle expressed as distance above the mean seal level of the world.
+- AGL: Altitude above Ground Level: the altitude of the vehicle above whatever is directly below it.
+- AH: Altitude above Home: the altitude distance above the Home position's altitude. This is what is usually displayed in the ground station and OSD as the vehicle's altitude.
```

Comment:
normally we call it "relative", I haven't seen the "AH" acronym before I think

Reply:
fixed